models/chainlink/arbitrum/chainlink_arbitrum_ccip_request_daily.sql

```diff
@@ -10,44 +10,53 @@
   )
 }}
 
-{% set incremental_interval = '7' %}
-{% set truncate_by = 'day' %}
-
 WITH
-  ccip_request_daily_meta AS (
-    SELECT
-      COALESCE(
-        cast(date_trunc('{{truncate_by}}', fulfilled.block_time) as date),
-        cast(date_trunc('{{truncate_by}}', reverted.block_time) as date)
-      ) AS "date_start",      
-      COALESCE(COUNT(fulfilled.token_amount), 0) as fulfilled_requests,
-      COALESCE(COUNT(reverted.token_amount), 0) as reverted_requests,
-      COALESCE(COUNT(fulfilled.token_amount), 0) + COALESCE(COUNT(reverted.token_amount), 0) as total_requests
-    FROM
-      {{ ref('chainlink_arbitrum_ccip_fulfilled_transactions') }} fulfilled
-      FULL OUTER JOIN {{ ref('chainlink_arbitrum_ccip_reverted_transactions') }} reverted ON
-        reverted.block_time = fulfilled.block_time AND
-        reverted.node_address = fulfilled.node_address
-    {% if is_incremental() %}
-      WHERE
-        fulfilled.block_time >= date_trunc('day', now() - interval '{{incremental_interval}}' day)
-        AND reverted.block_time >= date_trunc('day', now() - interval '{{incremental_interval}}' day)
-    {% endif %}
-    GROUP BY
-      1
-    ORDER BY
-      1
-  ),
-  ccip_request_daily AS (
-    SELECT
-      'arbitrum' as blockchain,
-      date_start,
-      cast(date_trunc('month', date_start) as date) as date_month,
-      fulfilled_requests,
-      reverted_requests,
-      total_requests
-    FROM ccip_request_daily_meta
-  )
+ethereum_agg AS (
+        SELECT
+            date_start,
+            (
+                SELECT
+                    COUNT(*)
+                FROM
+                    {{ ref('chainlink_arbitrum_ccip_fulfilled_transactions') }} eth
+                WHERE
+                    eth.date_start = date_series.date_start
+            ) AS fulfilled_requests,
+            (
+                SELECT
+                    COUNT(*)
+                FROM
+                    {{ ref('chainlink_arbitrum_ccip_reverted_transactions') }} rev
+                WHERE
+                    rev.date_start = date_series.date_start
+            ) AS reverted_requests
+        FROM
+            (
+                SELECT
+                    date_start
+                FROM
+            (
+                SELECT
```

Comment:
since this model is built incrementally on `date_start`, can't we apply incremental filters on both the sequence and the refs above?

Reply:
yeah ill add the incremental on those